funmi5/PropertyPro-Lite
# #gh-pages: Merge the UI files on gh-pages to the development branch

**What does this PR do?**
Merge the UI files on gh-pages to the development branch

How should this be manually tested/checked?
<a href= "https://funmi5.github.io/PropertyPro-Lite/UI/">Click on this link to check the web page</a>

What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2355287
